tgui/packages/tgui/interfaces/CloningPod.js

```diff
@@ -0,0 +1,151 @@
+import { useBackend } from '../backend';
+import {
+  Button,
+  ProgressBar,
+  Section,
+  Box,
+  Flex,
+  Icon,
+  NumberInput,
+} from '../components';
+import { Window } from '../layouts';
+
+export const CloningPod = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    biomass,
+    biomass_storage_capacity,
+    sanguine_reagent,
+    osseous_reagent,
+    organs,
+    currently_cloning,
+  } = data;
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section title="Liquid Storage">
+          <Flex height="25px" align="center">
+            <Flex.Item color="label" width="25%">
+              Biomass:{' '}
+            </Flex.Item>
+            <Flex.Item grow={1}>
```

Comment:
Another side note. If TGUI4 is implemented before this PR goes in, we have moved from using `<Flex>` to using `<Stack>` and this will need to be adjusted.